lib/validations.js

```diff
@@ -285,9 +282,15 @@ function validateLength(attr, conf, err, options) {
  * Numericality validator
  */
 function validateNumericality(attr, conf, err, options) {
-  if (nullCheck.call(this, attr, conf, err)) return;
+  if (nullCheck.call(this, attr, conf, err)) {
+    // If conf.allowBlank is set, we need to check typeof and isNaN again. Otherwise non numeric attributes can slip through
+    if (conf.allowBlank && typeof this[attr] === 'number' && isNaN(this[attr])) {
```

Comment:
The only test I see for `alloBlanck` is `should skip undefined attributes if conf.allowBlank is true` which passes. Please add test to cover the the failure for your logic here.